flow/designs/asap7/gcd/autotuner.json

```diff
@@ -8,22 +8,6 @@
         ],
         "step": 0
     },
-    "CORE_UTILIZATION": {
```

Comment:
Is this change in this PR intentional?

Reply:
Yes, it is. The rationale behind this removal is the that if the golden flow doesn't use it, autotuner will not use it either